contracts/fungible-tokens/andromeda-merkle-airdrop/src/contract.rs

```diff
@@ -162,6 +163,11 @@ pub fn execute_claim(
 
     // not expired
     let expiration = STAGE_EXPIRATION.load(deps.storage, stage)?;
+    let expiration = if let Some(expiration) = expiration {
```

Comment:
Can we do an existence check here instead of converting `None` to `Expriation::Never {}`?